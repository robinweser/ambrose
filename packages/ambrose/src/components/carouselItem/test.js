import React from 'react'

import CarouselItem from '.'

import createSnapshot from '../../testing/createSnapshot'

describe('<CarouselItem />', () => {
  it('renders correctly with default props', () => {
    const snapshot = createSnapshot(
      <CarouselItem maxWidth="250%">Hello</CarouselItem>
    )
    expect(snapshot).toMatchSnapshot()
  })
})
