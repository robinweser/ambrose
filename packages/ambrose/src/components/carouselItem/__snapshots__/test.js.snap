// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CarouselItem /> renders correctly with default props 1`] = `
".cls_0 {
  display: contents;
}
.cls_1 {
  box-sizing: border-box;
  flex-direction: column;
  flex-wrap: nowrap;
  flex-grow: 0;
  flex-shrink: 0;
  flex-basis: auto;
  align-items: stretch;
  display: flex;
  max-width: 250%;
  width: 90%;
  scroll-snap-align: center;
}
@media (min-width: 1024px) {
  .cls_1 {
    max-width: unset;
    width: auto;
  }
}


<div className=test-theme cls_0>
  <div className=cls_1>Hello</div>
</div>;
"
`;
